mbuild/packing.py

```diff
@@ -735,6 +845,14 @@ def solvate(
     # Apply edge buffer
     box_maxs = np.subtract(box_maxs, edge * 10)
     box_mins = np.add(box_mins, edge * 10)
+
+    # generate string of addl. packmol inputs given in packmol_args
+    packmol_commands = ""
+    if packmol_args:
+        check_packmol_args(packmol_args)
```

Comment:
If we are only going to support arg listed in `check_packmol_args`, I think we should move that list into the docs of this method (and then provide the PACKMOL link for additional reference). 

An alternative would be allow PACKMOL to handle the error, i.e., we can add a try-except block and throw the error/message that (some of) the keywords are not supported and the users should refer to the packmol docs for full list.

Reply:
I added the supported arguments to the doc strings for each method with some further explanation.